README.md

```diff
@@ -9,26 +9,26 @@ Reference application developed in the [Functional event-driven architecture: Po
 
 ## Table of contents
 
-- [trading](#trading)
-  - [Table of contents](#table-of-contents)
-  - [Web App](#web-app)
-  - [Overview](#overview)
-  - [Requirements](#requirements)
-    - [Running application](#running-application)
-  - [Services](#services)
-    - [Lib](#lib)
-    - [Domain](#domain)
-    - [Core](#core)
-    - [Feed](#feed)
-    - [Forecasts](#forecasts)
-    - [Processor](#processor)
-    - [Snapshots](#snapshots)
-    - [Alerts](#alerts)
-    - [WS Server](#ws-server)
-    - [Tracing](#tracing)
-    - [Tests](#tests)
-    - [X Demo](#x-demo)
-  - [Monitoring](#monitoring)
+* [trading](#trading)
+  * [Table of contents](#table-of-contents)
+  * [Web App](#web-app)
+  * [Overview](#overview)
+  * [Requirements](#requirements)
+    * [Running application](#running-application)
```

Comment:
this should be the only line change in the ToC, the rest should remain unchanged :)